bedrock/mozorg/templates/mozorg/contentservices/dmexco.html

```diff
@@ -0,0 +1,63 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contentservices/base.html" %}
+
+{% block page_title %}{{ _('@dmexco') }}{% endblock %}
+{% block body_id %}contentservices-dmexco{% endblock %}
+{% block body_class %}sand{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'contentservices' %}
+{% endblock %}
+
+{% block article %}
+  <h1 class="title-shadow-box">{{ self.page_title() }}</h1>
+
+  <div class="content-services-dmexco" itemscope itemtype="http://schema.org/Event">
+    <section>
+      <p itemprop="review">
+        <span itemprop="attendee">{{ _('Mozilla’s Content Services team') }}</span> {{ _('will be at <a href="http://dmexco.de/en/Home.html">dmexco 2015</a> to discuss our Tiles products and our vision for advancing user control on the Web.') }}
+        {{ _('We will be meeting with partners - both content partners and advertisers - who share our values and are looking for a future-facing media partner with global scale.') }}
+      </p>
+      <img itemprop="image" alt="dmexco logo" src="{{ static('img/mozorg/contentservices/dmexco_logo.jpg') }}">
```

Comment:
This page isn't being localized, but for consistency I have wrapped text. I will add it here  as well.